azd-remote-hosting-templates/Azure.Mcp.Server/aca-managed-identity/README.md

```diff
@@ -0,0 +1,61 @@
+# Azure MCP Server - ACA with Managed Identity
```

Comment:
did you mean to create an `azd-remote-hosting-templates` directory in the root of the `microsoft/mcp` repo?  Can you put this under `mcp/servers/Azure.Mcp.Server/samples`  (or `templates`?) instead - that way we keep the root directory clean to directories such as `{core/, docs/, eng/, servers/, tools/}`?

`mcp/servers/Azure.Mcp.Server/` already contains sub-folders for `{docs/, images/, src/, tests/}` so one more folder there specific to azmcp seems like the right location.

Reply:
nice, agree, this ^ structure is much cleaner. Yes, I'll go with `templates` or `adz-templates`. The templates directory name indicates these are azd templates without overcommitting to a broader "samples" structure. This keeps the scope focused and leaves room for later contents to be organized separately or merged with it in the future if needed.